content/en/tracing/trace_collection/library_config/java.md

```diff
@@ -389,47 +389,72 @@ Would produce the following result:
 
 {{< img src="tracing/setup/java/jmxfetch_example.png" alt="JMX fetch example"  >}}
 
-See the [Java integration documentation][9] to learn more about Java metrics collection with JMX fetch.
+See the [Java integration documentation][12] to learn more about Java metrics collection with JMX fetch.
 
-### B3 headers extraction and injection
+### Headers extraction and injection
 
-Datadog APM tracer supports [B3 headers extraction][10] and injection for distributed tracing.
+The Datadog APM Tracer supports [B3][13] and [W3C (TraceParent)][14] header extraction and injection for distributed tracing.
 
-Distributed headers injection and extraction is controlled by configuring injection/extraction styles. Currently two styles are supported:
+You can configure injection and extraction styles for distributed headers.
 
-- Datadog: `Datadog`
-- B3: `B3`
+The Java Tracer supports the following styles:
 
-Injection styles can be configured using:
+- Datadog: `datadog`
+- B3 Multi Header: `b3multi` (`B3` is deprecated)
```

Comment:
Maybe "`B3` alias is deprecated" to prevent ambiguity that B3 Multi Header itself is not deprecated.

Reply:
Good suggestion.